examples/todo/src/express.index.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {TodoListApplication} from './application';
+import {ApplicationConfig} from '@loopback/core';
+import {Request, Response} from '@loopback/rest';
+const express = require('express');
+const path = require('path');
+// const pEvent = require('p-event');
+
+export async function main(options: ApplicationConfig = {}) {
+  const app = new express();
+  const lbApp = new TodoListApplication(options);
+
+  // Mount LB4 as our REST API
+  await lbApp.boot();
+  lbApp.basePath('/api');
+
+  // Expose the front-end assets via Express, not as LB4 route
+  app.use(lbApp.requestHandler);
```

Comment:
Should it be the following?
```ts
app.use('/api', lbApp.requestHandler);
```

Reply:
It works when it's `app.use(lbApp.requestHandler);` because otherwise explorer fails because of `Fetch error: Failed to fetch http://127.0.0.1:3000/openapi.json`, and using `app.use('/api', lbApp.requestHandler);` along `lbApp.basePath('api');` makes the path for the LB app `/api/api` instead of `/api`.

Reply:
See discussion at https://github.com/strongloop/loopback-next/pull/2293

Reply:
Then I think I can work on this PR right now as is, and modify once that PR lands. What do you think?

Reply:
Sure.